conf/utd_ganymede.config

```diff
@@ -15,6 +20,28 @@ process {
   beforeScript = 'module load singularity/3.2.1'
   executor = 'slurm'
   queue = { task.memory >= 32.GB && task.cpu <= 12 ? 'Kim': task.memory <= 24.GB && task.cpu <= 8 ? 'smallmem' : 'genomics' }
+
+  withName:TRIMGALORE {
+    memory = 31.GB
+  }
+
+  withLabel:process_low {
+    cpus = { check_max( 2 * task.attempt, 'cpus' ) }
+    memory = { check_max( 12.GB * task.attempt, 'memory' ) }
+    time = { check_max( 6.h * task.attempt, 'time' ) }
+  }
+
+  withLabel:process_medium {
+    cpus = { check_max( 16 * task.attempt, 'cpus' ) }
+    memory = { check_max( 31.GB * task.attempt, 'memory' ) }
+    time = { check_max( 8.h * task.attempt, 'time' ) }
+  }
+
+  withLabel:process_large {
```

Comment:
This label doesn't exist in the nf-core pipeline template? Is it a custom one? Are the other default `label` breaking for you?

Reply:
Ah you're right it should be `high`. Those we're getting the right queue anyways, the main concern was bumping down the medium process to the specs that we have more nodes on.